DEPLOYMENT.md

```diff
@@ -39,6 +39,8 @@ git clone --recurse-submodules git@github.com:Shared-Reality-Lab/IMAGE-server.gi
 The default compose file within there (`docker-compose.yml`) is a useful base for testing. Each image will be downloaded from the Github Container Registry
 using the "unstable" tag.
 
+This can be brought up using `docker-compose up -d`.
+
 ## For Production
 
 For production, it is recommended to use "latest" and limit the number of running services for additional stability.
```

Comment:
suggest: "and limit the number of running services to fit your CPU, memory, and GPU limitations, for stability"